### PR TITLE
Integration test confirm

### PR DESCRIPTION
**Pull request by irubido**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "lint": "yarn lint:eslint && yarn lint:misc --check",
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
     "lint:changelogs": "yarn workspaces foreach --parallel --verbose run lint:changelog",
-    "test": "jest --passWithNoTests"
+    "test": "yarn workspaces foreach run test"
```
Comment (mpetrunic) [inline package.json]:
```suggestion
    "test": "yarn workspaces foreach --verbose run test"
```

File: packages/confirm/test/integration/index.spec.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      'confirm',
+      ['Test-prompt', 'Test-description', 'Test-textAreaContent'],
+    );
+
+    await dappeteer.snaps.acceptDialog();
+    const result = await resultPromise;
+
+    expect(result).toBe(true);
+  });
+
```
Comment (BeroBurny) [inline packages/confirm/test/integration/index.spec.ts:45]: you should add a case when confirmation is rejected

**Review by BeroBurny — COMMENTED** [review]
over all looks good

File: packages/confirm/jest.config.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (mpetrunic) [inline packages/confirm/jest.config.js]: Copy https://github.com/MetaMask/metamask-module-template/blob/main/jest.config.js into root of repo. add property `"root": true` and extend it in every package

File: tsconfig.json [hunk 4]
```diff
@@ -3,9 +3,9 @@
     "declaration": true,
     "esModuleInterop": true,
     "inlineSources": false,
-    "module": "ES6",
+    "module": "CommonJS",
```
Comment (mpetrunic) [inline tsconfig.json]: don't think you should be changing that.

File: .github/workflows/build-test.yml [hunk 5]
```diff
@@ -32,7 +32,8 @@ jobs:
       - run: yarn --immutable
       - run: yarn build
       - run: yarn lint
-      - run: yarn test
+      - run: yarn add playwright -D
```
Comment (mpetrunic) [inline .github/workflows/build-test.yml]: remove, needs to be another way

File: .yarnrc [hunk 6]
```diff
@@ -0,0 +1 @@
+ignore-scripts true
```
Comment (mpetrunic) [inline .yarnrc]: remove this file

File: packages/confirm/package.json [hunk 7]
```diff
@@ -52,6 +53,7 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.4.0",
     "jest": "^26.4.2",
+    "playwright": "^1.27.1",
```
Comment (mpetrunic) [inline packages/confirm/package.json]: maybe add only to root package.json?

**Comment by mpetrunic** [comment]
tests are passing, but there is a type error with closing browser

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**